app/code/Magento/MediaContentApi/Api/GetContentWithAssetInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\MediaContentApi\Api;
+
+/**
+ * Get media content list which is used with the specified media asset
+ * @api
+ */
+interface GetContentWithAssetInterface
+{
+    /**
+     * @param int $assetId
+     *
+     * @return array
```

Comment:
what's the format of return result ? Not looking at implementation it's not clear

Reply:
It's ContentIdentityInterface[] (introduced data interface), the code is updated